code/modules/clothing/spacesuits/rig/modules/utility.dm

```diff
@@ -418,29 +428,14 @@
 	interface_name = "mop projector"
 	interface_desc = "A mop that can be deployed from the hand of the wearer."
 
-	usable = 0
+	usable = 1
 	selectable = 1
 	toggleable = 1
-	use_power_cost = 0
+	use_power_cost = 5
 	active_power_cost = 0
 	passive_power_cost = 0
 
-	gun = /obj/item/weapon/reagent_containers/spray/cleaner
-
-//obj/item/weapon/reagent_containers/spray/cleaner
-//	spary =
-
-/obj/item/rig_module/mounted/engage(atom/target)
-
-	if(!..())
-		return 0
-
-	if(!target)
-		gun.attack_self(holder.wearer)
-		return 1
-
-	gun.Fire(target,holder.wearer)
-	return 1
+	gun_type = /obj/item/weapon/gun/energy/temperature/mounted
```

Comment:
If it uses a gun, shouldn't it go with the other mounted gun modules?

Reply:
Theoretically, though its main 'purpose' is the mop.